pcdsdevices/spectrometer.py

```diff
@@ -307,6 +309,13 @@ def calc_lightpath_state(self, **kwargs) -> LightpathState:
         )
 
 
+class FZPStates(TwinCATStatePMPS):
+    """
+    FZP ND States Setup
```

Comment:
I know this is fresnel zone plate, but does everyone else? ~Macro~ acronym expansion in docstring 🙏

Reply:
You are 100% correct